cypress/e2e/tenure-comments.feature

```diff
@@ -1,8 +1,12 @@
-@authentication
 @comments
+@authentication
 @common
-@processes
 @root
+@tenure
+@search
+@worktray
+@processes
+@personal-details
```

Comment:
When any of these receive changes.... _(say person details displays an extra data field, or search gets an extra button added, etc.)_ 

... should they trigger a feature spec that's testing the adding of comments under a tenure?